packages/cli/src/lib/config/sessionConfig.ts

```diff
@@ -42,16 +41,15 @@ export class SessionConfig extends ConfigFileDefault<SessionConfigEntity> {
     setLastHubId(lastHubId: string): boolean {
         return this.setEntry("lastHubId", lastHubId);
     }
-    protected validateEntry(key: string, value: any): boolean | null {
+    protected validateEntry(key: string): boolean | null {
         switch (key) {
             case "lastPackagePath":
             case "lastSpaceId":
             case "lastSequenceId":
             case "lastHubId":
+            case "lastInstanceId":
             case "sessionId":
                 return null;
```

Comment:
check if lastInstanceId is "" or has required length

Reply:
@patuwwy  
Instance ID is checked [here](https://github.com/scramjetorg/transform-hub/pull/966/files#diff-49139fa83ab4d9b3443fcffc2158a1f67af14baa9622bf060cd5c92c32a3343bR930) during sequence start, as a matter of fact whole payload is being checked here. We use a method [isStartSequenceEndpointPayloadDTO() ](https://github.com/scramjetorg/transform-hub/pull/966/files#diff-8da181dfd902335d0ab6991c5ad6c0edc82ebdd58ec454ab5320cd0f61b8adb8R25) that `instanceId` checks length, type and if it's not `undefined`. 
Do we need to check it again in the session config?